dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -1046,7 +1046,7 @@ BEGIN
     PUSHBUTTON "&Reguli de apel", 1108, 193, 110, 64, 14
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 7, 130, 250, 1
     DEFPUSHBUTTON "&Apelează", 1590, 6, 136, 60, 14
-    PUSHBUTTON "A&nulează", 1591, 69, 136, 60, 14
+    PUSHBUTTON "&Revocare", 1591, 69, 136, 60, 14
```

Comment:
This change does introduce an accelerator collision with the line
PUSHBUTTON "&Reguli de apel", 1108, 193, 110, 64, 14
Check your Romanian Windows to find out which one needs to be changed.